internal/command/testversions/testprovider3

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
I would probably suggest that you name this provider to indicate that it will fail (`testprovider3` -> `testprovider-noversionsupport`)